README.md

```diff
@@ -6,17 +6,15 @@ Ballerina WebSub Library
   [![Github issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-standard-library/module/websub.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-standard-library/labels/module%2Fwebsub)
   [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-The WebSub library is one of the standard library modules of the<a target="_blank" href="https://ballerina.io/"> Ballerina</a> language.
+The WebSub package is one of the standard library packages of the <a target="_blank" href="https://ballerina.io/">Ballerina</a> language.
 
-This library provides the capability to easily introduce publishers and subscribers that are WebSub-compliant. The ballerina/websub library comes with an in-built WebSub Hub service, which can be brought up by publishers that need to bring up a hub.
-
-For example demonstrations of the usage, go to [Ballerina By Examples](https://ballerina.io/swan-lake/learn/by-example/websub-internal-hub-sample.html).
+This package provides the capability to easily introduce publishers and subscribers that are WebSub-compliant. The ballerina/websub package comes with an in-built WebSub Hub service, which can be brought up by publishers that need to bring up a hub.
```

Comment:
Is this statement correct? Does this package comes with an in-built websubhub service or is it in a separate package (websubhub) ?

Reply:
* Updated the document properly.